package.json

```diff
@@ -6,14 +6,15 @@
   "type": "module",
   "exports": {
     ".": "./dest/index.js",
-    "./eslint": "./.eslintrc.cjs",
-    "./prettier": "./.prettierrc.json"
+    "./eslint": "./estlint-config/index.cjs",
+    "./prettier": "./prettier-config/index.cjs",
+    "./json-rpc": "./json-rpc/src/index.js"
```

Comment:
Should just be exported from `src/index.ts`?

Reply:
Your call :) Sure